lib/open_api_2_json_schema.rb

```diff
@@ -147,4 +164,12 @@ def clean_required(required, properties)
 
     properties.slice(*required).keys
   end
+
+  def move_hash_value_one_upper_level(hash, upper_key)
```

Comment:
Do we need this?

Reply:
The ref-schema might have the top-level `schema` key, which won't work when trying on `react-jsonschema-form`

Reply:
Resolved due to private discussion, will update the code soon